packages/protobuf/src/messages.ts

```diff
@@ -375,6 +375,8 @@ export type TxOutputType =
           orig_index?: number;
           payment_req_index?: number;
       }
+    // NOTE: the type was loosened for compatibility (issue #10474)
+    // It is not originally intended to use address instead of address_n with change output
```

Comment:
why was it removed, and how was it added? :D

Reply:
I added it manually probably